doc/howtos/ip-address-config.rst

```diff
@@ -0,0 +1,103 @@
+.. _ip-address-config:
+
+Configuring an IP Address in the Ostro |trade| OS
+######################################################
+
+The Ostro Project uses Connection Manager (ConnMan), a Linux daemon and command-line utility, for managing internet connections 
+within embedded devices. 
```

Comment:
Although correct I'd rephrase to put some emphasis on the DBus API and that it is official way to talk to connman. Then you can follow using connmanctl, then we add some DBus call that does the same, people can then use Python DBus or something like that in real products instead of trying to call system() on connmanctl

Soletta and nodejs will have nice C/FBP/JS version, so we can refer to them later.